hca/util/__init__.py

```diff
@@ -240,9 +243,8 @@ def swagger_spec(self):
             else:
                 swagger_filename = base64.urlsafe_b64encode(swagger_url.encode()).decode() + ".json"
                 swagger_filename = os.path.join(self.config.user_config_dir, swagger_filename)
-            is_cached = os.path.exists(swagger_filename)
-            if (not is_cached) or (is_cached and
-                                   self._get_days_since_last_modified(swagger_filename) >= self._spec_valid_for_days):
+            if (("swagger_filename" not in self.config) and ((not os.path.exists(swagger_filename)) or
```

Comment:
do you want the auto-update to happen if someone specifies `swagger_filename` in the config?

Reply:
No, if someone specifies `swagger_filename` in the config, we assume that they are using a local copy of the Swagger file.  We wouldn't want to overwrite their local copy and only want to update to the latest Swagger file in the user config directory `~/hca/.config`.